packages/serverless-components/nextjs-cdk-construct/examples/basic/package.json

```diff
@@ -33,20 +31,9 @@
     "typescript": "4.3.5"
   },
   "dependencies": {
-    "@aws-cdk/aws-certificatemanager": "^1.117.0",
-    "@aws-cdk/aws-cloudfront": "^1.117.0",
-    "@aws-cdk/aws-cloudfront-origins": "^1.117.0",
-    "@aws-cdk/aws-iam": "^1.117.0",
-    "@aws-cdk/aws-lambda": "^1.117.0",
-    "@aws-cdk/aws-lambda-event-sources": "^1.117.0",
-    "@aws-cdk/aws-logs": "^1.117.0",
-    "@aws-cdk/aws-route53": "^1.117.0",
-    "@aws-cdk/aws-route53-targets": "^1.117.0",
-    "@aws-cdk/aws-s3": "^1.117.0",
-    "@aws-cdk/aws-s3-deployment": "^1.117.0",
-    "@aws-cdk/aws-sqs": "^1.117.0",
-    "@aws-cdk/core": "1.117.0",
-    "@sls-next/cdk-construct": "^3.2.0",
+    "@sls-next/cdk-construct": "link:../../",
+    "aws-cdk-lib": "2.0.0",
+    "constructs": "10.0.0",
```

Comment:
May I wish the versions not be pinned to the patch level? Otherwise I'd have to override dependencies to overcome TypeScript strict errors.

Reply:
@shishkin 
Fixed at 0a62b3c. Could you please confirm that I am not mistaken?